src/main/java/com/kashdeya/tinyprogressions/world/WorldGen.java

```diff
@@ -1,66 +1,39 @@
 package com.kashdeya.tinyprogressions.world;
 
+import com.google.common.collect.Lists;
+import net.minecraft.util.RegistryKey;
+import net.minecraft.util.registry.WorldGenRegistries;
+import net.minecraft.world.World;
+import net.minecraft.world.biome.Biome;
+import net.minecraft.world.biome.BiomeGenerationSettings;
+import net.minecraft.world.gen.GenerationStage;
+import net.minecraft.world.gen.feature.ConfiguredFeature;
+import net.minecraftforge.fml.common.ObfuscationReflectionHelper;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import java.util.function.Supplier;
+
+// Ender Ore
+
 public class WorldGen {
-//
```

Comment:
A lot of the commented code is things that need to get implemented. If we can keep until all these features are added I would rather do that. I will go in and clean out the things that are no longer needed.

Reply:
Will do tomorrow